src/Synapse/Synapse/help/New-AzSynapseManagedPrivateEndpoint.md

```diff
@@ -79,7 +79,16 @@ Accept wildcard characters: False
 ```
 
 ### -DefinitionFile
-The JSON file path.
+The JSON file path. Below is the json file structure example:
+```json
+{
+	"name": "ContosoManagedPrivateEndpoint",
+	"properties": {
+		"privateLinkResourceId": "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/contosoResourceGroup/providers/Microsoft.Storage/storageAccounts/contosoStorageAccount",
+		"groupId": "file"
+	}
+}
+```
```

Comment:
Please update the HelpMessage of DefinitionFile for this command. If not, I suppose these help messages will be lost if you regenerate this help doc.

Reply:
Another way to add sample file in example is adding it as comments in script, see [Set-AzKeyVaultKeyRotationPolicy]( https://github.com/Azure/azure-powershell/blob/main/src/KeyVault/KeyVault/help/Set-AzKeyVaultKeyRotationPolicy.md#example-1-sets-key-rotation-policy-by-json-file)

Reply:
added the sample json to example1 as a comment.